File: Fundamentals/Lecture-4-Exercise/odd_and_even_sum.py
a = input()
b = len(a)
odd_sum = 0
even_sum = 0
for i in map(int,a):
    if i % 2 == 0:
        even_sum += i
    else:
        odd_sum += i
    
print(f"Odd sum = {odd_sum}, Even sum = {even_sum}")

File: JS-FRONT-END/Functions_and_Statements_Exercise/oddAndEvenSum.js
function solve(number) {
    let numberString = String(number).split('')
    let splittedNumber = numberString.map(function (item) {
        return parseInt(item)
    });
    // console.log(splittedNumber)
    let oddSum = 0;
    let evenSum = 0;

    for (const numberElement of splittedNumber) {
        if (Number((numberElement) % 2 === 0)){
            evenSum += numberElement
        }else{
            oddSum += numberElement
        }
    }
    console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`)
}

solve(3495892137259234)